.github/workflows/CI.yml

```diff
@@ -0,0 +1,193 @@
+name: CI
+on:
+  # always execute docker build when something is pushed to main/master or release-* branches
+  push:
+    branches:
+      - "master"
+      - "main"
+      - "release-*"
+  # in addition, execute for pull requests to those branches
+  pull_request:
+    branches:
+      - "master"
+      - "main"
+      - "release-*"
+
+  workflow_dispatch: # run integration tests only when triggered manually
```

Comment:
I think the comment is not correct

Reply:
Thanks, will update this also in service-template.